server/templates/labelVideo.html

```diff
@@ -16,144 +16,192 @@
 -->
 <html>
 <head>
-  <link rel="stylesheet"
-      href="https://fonts.googleapis.com/icon?family=Material+Icons">
-  <link rel="stylesheet"
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap">
-  <link rel="stylesheet" type="text/css"
-      href="https://storage.googleapis.com/{{project_id}}/static/css/styles.css?ignoreCache={{time_time}}">
-  <script type="text/javascript"
-      src="https://storage.googleapis.com/{{project_id}}/compiled/js/fmltc.js?ignoreCache={{time_time}}">
-  </script>
-  <title>Video Frame Labeling</title>
+  <title>FIRST Machine Learning Toolchain</title>
```

Comment:
@cmacfarl Do we want to put in a different placeholder title here? Maybe FIRST Object Detection Training Tool?

Reply:
I would prefer this. The initials alone of the current one are... questionable. FODTT while it doesn't roll of the tongue it doesn't invite memes. :)